tests/cli-v2.rs

```diff
@@ -808,6 +809,61 @@ fn remove_target_missing_update_hash() {
     });
 }
 
+// Issue #1777
+#[test]
+fn warn_about_and_remove_stray_hash() {
+    setup(&|config| {
+        let mut hash_path = config.rustupdir.join("update-hashes");
+        fs::create_dir_all(&hash_path).expect("Unable to make the update-hashes directory");
+
+        hash_path.push(for_host!("nightly-{}"));
+        println!("Update-hash path: {}", hash_path.display());
```

Comment:
This println seems superfluous debugging.  Probably remove it now you're satisfied?